src/python/gudhi/persistence_graphical_tools.py

```diff
@@ -45,6 +45,13 @@ def __min_birth_max_death(persistence, band=0.0):
     return (min_birth, max_death)
 
 
+def _array_handler(a):
+    if isinstance(a, np.ndarray): 
```

Comment:
```suggestion
    if isinstance(a[0][1], np.float64) or isinstance(a[0][1], float):
```
Historical accepted format is tuples(dimension, tuple(birth, death)).
What I propose would allow tuple(birth, death) but also numpy.array of shape (n,2).